book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+(rr-rdm-elns)=
+# Electronic Lab Notebooks
+
+An Electronic Laboratory Notebook (commonly known as an ELN or a digital lab notebook) is a software system designed to help you document and maintain reproducibility of your research and share information more easily.
+ELNs enable researchers to organize and store experimental procedures, protocols, plans, notes, data, and even unfiltered interpretations using their computer or mobile device.
+They can be a digital analogue to the paper notebook most researchers keep.
+ELNs can offer several advantages over the traditional paper notebook in documenting research during the active phase of a project, including; searchability within and across notebooks, secure storage with multiple redundancies, remote access to notebooks, and the ability to easily share notebooks among team members and collaborators.
+ELNs make the practice of {ref}`open notebooks <rr-open-notebooks>` [{term}`def<Open Notebooks>`] practical in a way that it is not for paper lab books.
+However, it is important when choosing an ELN solution to avoid giving up the advantages offered by a paper lab notebook.
+
+(rr-rdm-elns-procon)=
+## Quick Pros & Cons of ELNs
+
+Electronic Lab Notebooks may provide, among other things, the following **functionalities**:
+
+* A text editor with similar functions as a paper notebook
+* A search function
+* Secure storage and back up (with distinct failure modes from paper notebooks)
+* Tools for working with tabular data (calculations and formatting of tables and graphs)
+* Templates for documenting standard procedures
+* Built-in Laboratory Inventory Management Systems (LIMS) functionality for managing and documenting samples, reagents, and apparatus, or integration with separate dedicated LIMS.
+* Collaboration tools for sharing experimental information
+* Some ELNs will allow you to comply with standards and regulations because of their certification processes
+
+ELNs also have their **limitations**: 
+
+* **Costs**: Most ELN solutions can only be used through paid plans, or the free plans have reduced functionalities in the number of users, storage space or file sizes.
+Your lab may also not have access to tablets or pens that would make the use of ELNs easier.
+Check if your institution is offering a solution that you can use without additional costs.
+* **File format**: Always check if files can be exported in your preferred file formats to avoid format lock-in.
+Some ELNs also have API's that allow integration with other software and workflows. 
+* **Vendor-lock-in**: Once you're using a certain solution you may become dependent on it.
```

Comment:
Maybe a reference to the lock-in section below?

Reply:
Addressed in [7322151](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/7322151f466607fe81a458c5ef56e89bc3f16dea)